### PR TITLE
Converts Every Other Shuttle to /shuttle/ areas

### LINKED ISSUE #54: Maps are FUCKED
## Reproduction:

All maps had to be updated in https://github.com/Willardstation/tg-voidcrew/pull/50 but for the sake of getting the game working, only Box was done.

All other maps needs to have custom areas made (look at box in that PR) and the map fixed.

Also also some maps (including box) have firelocks under windows next to space. This causes endless and unstoppable firelock hell, we need to remove firelocks from under the damn windows/shutters to space.

### PR DESCRIPTION
**Pull request by unit0016**
## About The Pull Request
Converts every shuttle to the new /area/ system, one by one. This was agony and way bigger a project than I was expecting it to be. Help.

Notable Additional Changes:
- The HONK Class and Lamia class have received unique area names that I thought would be cute.
- Magic power is gone from the Hyena. Yeah. Really.

**FUCKY WUCKYS**:
- Docking port variables aren't all up to snuff. Some shuttles fly sideways, otherwise turn when they really shouldn't and lag the whole ass server (Metis), etc. It'd be a nightmare to sort through and I'm happy with just having functionality.
## Why It's Good For The Game
*how can the ship fly if ship fly fly ship the fly?* 
**- A great philosopher from Gorlex, shortly before a history-defining tragedy strikes their people**

None of these do any other major additional fixes nor contribute towards #18, and need additional love asides from "IT CAN RUN, SHIP IT."

Closes #54 
## Changelog
:cl:
fix: Every [shuttle]'s here!
/:cl:

### CODE REVIEW

**Comment by unit0016** [comment]
I'm back awake, drafting to hammer out the rest

**Comment by unit0016** [comment]
Pausing here for a bit to take a break, Fly me to the Moon (Climax Mix) can only take a girl so far. Only 7 left to go.

**Comment by unit0016** [comment]
All NT/Independent shuttles done. Taking another break and I'll knock out the remaining syndie ships

**Comment by unit0016** [comment]
Fuck, Hyena refuses to load.

**Comment by unit0016** [comment]
phalanx is really fucked up by the way, free AIO telecomms machine (which also spreads comms to every z-level), bunch of other gunk too.

File: tgstation.dme [hunk 1]
```diff
@@ -4914,7 +4913,7 @@
 #include "voidcrew\mapping\shuttles\irish.dm"
 #include "voidcrew\mapping\shuttles\kilo.dm"
 #include "voidcrew\mapping\shuttles\lamia.dm"
-#include "voidcrew\mapping\shuttles\luxembourg.dm"
+#include "voidcrew\mapping\shuttles\libertatia.dm"
```
Comment (JohnFulpWillard) [inline tgstation.dme:4916]: libertatia was missing from the dme and lints didn't catch it?

File: tgstation.dme [hunk 2]
```diff
@@ -4923,13 +4922,15 @@
...
-#include "voidcrew\mapping\shuttles\spitfire.dm"
 #include "voidcrew\mapping\shuttles\nanotrasen\bead.dm"
 #include "voidcrew\mapping\shuttles\nanotrasen\phalanx.dm"
 #include "voidcrew\mapping\shuttles\nanotrasen\thunderbird.dm"
 #include "voidcrew\mapping\shuttles\syndicate\blackbeard.dm"
+#include "voidcrew\mapping\shuttles\syndicate\boyardee_b.dm"
 #include "voidcrew\mapping\shuttles\syndicate\geneva.dm"
 #include "voidcrew\mapping\shuttles\syndicate\hyena.dm"
+#include "voidcrew\mapping\shuttles\syndicate\luxembourg.dm"
+#include "voidcrew\mapping\shuttles\syndicate\spitfire.dm"
```
Comment (JohnFulpWillard) [inline tgstation.dme:4933]: I think originally I wanted to limit NT/Syndicate ships to 3 each, now there's 3 NT and 6 Syndicate

File: tgstation.dme (hunk 1 above)
Comment (unit0016) [reply tgstation.dme:4916]: yeah, it was. no idea why, frankly slightly concerning.

File: tgstation.dme (hunk 2 above)
Comment (unit0016) [reply tgstation.dme:4933]: That's actually 100% fair - I'll repath to what they were before as the alternative is an arms race between the two, mapping wise - and although that'd be fitting, scope creep bad ackshually.

That being said, 3 of each means they've gotta have their best feet forward, so I guess I have my next remap targets.

**Review by JohnFulpWillard — COMMENTED** [review]
I accidentally started a review instead of leaving single comment then forgot to leave it ☹️

File: voidcrew/mapping/shuttles/boyardee.dm [hunk 3]
```diff
@@ -32,3 +32,65 @@
 			slots = 2,
 		),
 	)
+
+/// DOCKING PORT ///
+
+/obj/docking_port/mobile/voidcrew/bogardee // I'm not correcting this, it's funny.
+	name = "Boyardee-Class Type Entertainement Vessel"
+	area_type = /area/shuttle/voidcrew/bogardee
+	callTime = 25 SECONDS
```
[the remark this answers is not in the document]
Comment (unit0016) [reply voidcrew/mapping/shuttles/boyardee.dm:41]: I'm gonna go back in, move this up, and audit port_dir to the best of my abilities.

**Comment by unit0016** [comment]
OKAY SO
I fucked up the other docking port variables something AWFUL. Their directions are all fucked up (really proving how even with the easiest version of shuttlecode imaginable i'm still a dumbass with it), though, again, they fly around just fine - though in the process of attempting to address that I had a realization.

*Roundstart shuttles still don't load.*

I don't think I'll be able to fix dirs for this PR, but that's a bigger issue-worthy thing, that.

**Comment by JohnFulpWillard** [comment]
I don't think every docking port needs to be a new subtype, just putting the all-shared stuff on 'voidcrew' parent is fine, especially dirs which I think are important to be a mapping thing that can easily be changed if the map itself is ever rotated

**Comment by unit0016** [comment]
> I don't think every docking port needs to be a new subtype, just putting the all-shared stuff on 'voidcrew' parent is fine, especially dirs which I think are important to be a mapping thing that can easily be changed if the map itself is ever rotated

The linked area var is the reason they've been made the way they are, because that var really isn't the kind you want defined on a mapping level.
That being said, I'll transition the dirs back to being a mapping thing.

**Comment by GuiltyNeko** [comment]
> phalanx is really fucked up by the way, free AIO telecomms machine (which also spreads comms to every z-level), bunch of other gunk too.

AIO comms isn't multi-z. I know this because I tried it.

**Comment by unit0016** [comment]
> AIO comms isn't multi-z. I know this because I tried it.

https://github.com/Willardstation/tg-voidcrew/blob/b8cd75ce7c58db5a0f82f2f3daff4c6f71efaa43/code/game/machinery/telecomms/machines/allinone.dm#L32-L34

Minor correction: it doesn't on the Phalanx, as that functionality is locked behind `intercept` being set to `1`. This functionality, which you should've known existed considering you've come here to proclaim you've tried it, is currently exclusive to the Syndicate Infiltrator (Nukie Shuttle) as a gameplay means to take out (or hijack) SyndiComms during Nuke Ops rounds.

**Comment by GuiltyNeko** [comment]
> Minor correction: it doesn't on the Phalanx, as that functionality is locked behind `intercept` being set to `1`. This functionality, which you should've known existed considering you've come here to proclaim you've tried it, is currently exclusive to the Syndicate Infiltrator (Nukie Shuttle) as a gameplay means to take out (or hijack) SyndiComms during Nuke Ops rounds.

https://github.com/Willardstation/tg-voidcrew/blob/b8cd75ce7c58db5a0f82f2f3daff4c6f71efaa43/code/game/machinery/telecomms/machines/allinone.dm#L6-L13
Minor correction: The syndicate AIO is varedited with that set. Furthermore, it overwrites the listening frequencies to only be syndicate. By default, intercept is off. With intercept off... 

https://github.com/Willardstation/tg-voidcrew/blob/b8cd75ce7c58db5a0f82f2f3daff4c6f71efaa43/code/game/machinery/telecomms/machines/allinone.dm#L29

It only works on the same z level. No amount of relays will allow it to extend to other z-levels, either. It is solely the bus/processor/broadcaster/receiver in one machine.

**Comment by unit0016** [comment]
That was, in fact, the point of my correction, and what it said, yes.

You came here to pick a fight, aren't a maintainer, and *forked **Merchant** of all repos*, so that's everything I need to know in a neat little bow. Old reliable time.

**Comment by JohnFulpWillard** [comment]
we should simply not use syndicate radio frequencies, any sort of permanent radio channel i think is not great and people should have to build it themselves

**Comment by unit0016** [comment]
There's a few syndicomm headsets scattered around a few ships right now, but those're easy enough to remove.

**Comment by JohnFulpWillard** [comment]
> There's a few syndicomm headsets scattered around a few ships right now, but those're easy enough to remove.

would be nice

**Pull request merged**